### PR TITLE
chore(CHANGELOG): update to v3.4.0

### PR DESCRIPTION
**Pull request by oguzeroglu**

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
I think that should be mention as well: https://github.com/camunda/camunda-modeler/commit/26197739b00b731d5b709eab479151d545bc453e

Our new client extension point 🚀

**Comment by barmac** [comment]
We usually also include the features which are shipped with the updated libraries, e.g. https://github.com/camunda/camunda-modeler/blob/develop/CHANGELOG.md#bpmn

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,30 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
 
+## 3.4.0    
+
+#### General    
+
+* `FEAT`: add notifications ([#1505](https://github.com/camunda/camunda-modeler/issues/1505))
+* `FEAT`: add extension point for ui plugins ([#1490](https://github.com/camunda/camunda-modeler/issues/1490))
+* `FEAT`: add stable api to access workspace configuration fron plug-ins ([#1490](https://github.com/camunda/camunda-modeler/issues/1425))
+* `FIX`: complete direct editing on save operation ([#1473](https://github.com/camunda/camunda-modeler/issues/1473))
+* `FIX`: modal autofocus ([#1489](https://github.com/camunda/camunda-modeler/pull/1489))
```
Comment (pinussilvestrus) [inline CHANGELOG.md]: fix autofocusing for modals?

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,30 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 
 ___Note:__ Yet to be released changes appear here._
 
+## 3.4.0    
+
+#### General    
+
+* `FEAT`: add notifications ([#1505](https://github.com/camunda/camunda-modeler/issues/1505))
+* `FEAT`: add extension point for ui plugins ([#1490](https://github.com/camunda/camunda-modeler/issues/1490))
+* `FEAT`: add stable api to access workspace configuration fron plug-ins ([#1490](https://github.com/camunda/camunda-modeler/issues/1425))
```
Comment (pinussilvestrus) [inline CHANGELOG.md]: "from"

I'd say

"allow access to workspace configurations for plugins and files"

+ Ensure you reference the correct issue number (1425)

File: CHANGELOG.md (hunk 1 above)
Comment (oguzeroglu) [reply CHANGELOG.md]: It's this one: https://github.com/camunda/camunda-modeler/pull/1489

Should I remove it?

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,30 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 
 ___Note:__ Yet to be released changes appear here._
 
+## 3.4.0    
+
+#### General    
+
+* `FEAT`: add notifications ([#1505](https://github.com/camunda/camunda-modeler/issues/1505))
+* `FEAT`: add extension point for ui plugins ([#1490](https://github.com/camunda/camunda-modeler/issues/1490))
```
Comment (pinussilvestrus) [inline CHANGELOG.md]: Maybe we could add this feature as well:  https://github.com/camunda/camunda-modeler/commit/c1716540efde646d1b74e1827ae0a78fb02cb1ab

Maybe you can list all of these events in one section (so e.g. <app.activeTabChanged>...). This way we would get rid of the BPMN section, which only includes two of these events

File: CHANGELOG.md (hunk 1 above)
Comment (pinussilvestrus) [reply CHANGELOG.md]: Was just wording recommendation :) We should include it.

Comment (barmac) [reply CHANGELOG.md]: It's okay to reword the commit message when you prepare a changelog update.

File: CHANGELOG.md (hunk 3 above)
Comment (oguzeroglu) [reply CHANGELOG.md]: BPMN section now also contains elements fixed with bpmn-js release

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,35 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
+* `FEAT`: allow access to workspace configurations for plugins and files ([#1425](https://github.com/camunda/camunda-modeler/issues/1425))
+* `FEAT`: provide foundation for cawemo extension ([#1496](https://github.com/camunda/camunda-modeler/issues/1496))
```
Comment (pinussilvestrus) [inline CHANGELOG.md]: We don't want to include this, it's just internal.

File: CHANGELOG.md (hunk 3 above)
Comment (pinussilvestrus) [reply CHANGELOG.md]: Maybe let's move this one to a single section:

```js
#### Plugins

* `FEAT`: add extension point for ui plugins ([#1490](https://github.com/camunda/camunda-modeler/issues/1490))
* Introduce several application events to hook into:
    * <tab.saved> ([#1498](https://github.com/camunda/camunda-modeler/pull/1498))
    * <tab.activeSheetChanged> ([`403afc`](https://github.com/camunda/camunda-modeler/commit/403afc920cf6c745816c3cd456baeb99830b25ed))
    * <app.activeTabChanged> ([`403afc`](https://github.com/camunda/camunda-modeler/commit/403afc920cf6c745816c3cd456baeb99830b25ed)) 
    * <bpmn.modeler.configure> ([#1499](https://github.com/camunda/camunda-modeler/issues/1499))
    * <bpmn.modeler.created> ([#1500 ](https://github.com/camunda/camunda-modeler/issues/1500))
````

Then we can remove these references  from the

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,41 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
 
+## 3.4.0    
+
+#### General    
+
+* `FEAT`: add notifications ([#1505](https://github.com/camunda/camunda-modeler/issues/1505))
+* `FEAT`: allow access to workspace configurations for plugins and files ([#1425](https://github.com/camunda/camunda-modeler/issues/1425))
+* `FIX`: complete direct editing on save operation ([#1473](https://github.com/camunda/camunda-modeler/issues/1473))
+* `FIX`: fix autofocusing for modals ([#1489](https://github.com/camunda/camunda-modeler/pull/1489))
+* `CHORE`: add example for plugin config ([#1512](https://github.com/camunda/camunda-modeler/issues/1512))
```
Comment (pinussilvestrus) [inline CHANGELOG.md]: maybe let's remove this one.

**Pull request merged**